staging/src/k8s.io/code-generator/generate-groups.sh

```diff
@@ -49,44 +49,44 @@ shift 4
 (
   # To support running this script from anywhere, we have to first cd into this directory
   # so we can install the tools.
-  cd $(dirname "${0}")
-  go install ${GOFLAGS:-} ./cmd/{defaulter-gen,client-gen,lister-gen,informer-gen,deepcopy-gen}
+  cd "$(dirname "${0}")"
+  go install ./cmd/{defaulter-gen,client-gen,lister-gen,informer-gen,deepcopy-gen}
 )
 
 function codegen::join() { local IFS="$1"; shift; echo "$*"; }
 
 # enumerate group versions
 FQ_APIS=() # e.g. k8s.io/api/apps/v1
 for GVs in ${GROUPS_WITH_VERSIONS}; do
-  IFS=: read G Vs <<<"${GVs}"
+  IFS=: read -r G Vs <<<"${GVs}"
```

Comment:
why this?

Reply:
it hits https://github.com/koalaman/shellcheck/wiki/SC2162

Reply:
ok, looks good